test/dependencies/go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/ipld/go-ipld-prime v0.0.4-0.20200503082126-7e0619f3a984
 	github.com/jbenet/go-random v0.0.0-20190219211222-123a90aedc0c
 	github.com/jbenet/go-random-files v0.0.0-20190219210431-31b3f20ebded
-	github.com/libp2p/go-libp2p v0.9.6
+	github.com/microcosm-cc/bluemonday@1.0.16
```

Comment:
What the heck? Replacing go-libp2p with a HTML sanitizer? That's... interesting!

If this is supposed to be a joke, it's not a bad one. I laughed out loud when I saw this!

Reply:
One a more serious note, go-libp2p v0.9.6 is reeeeaaaally old. It was released in June 2020.

Reply:
These are testing dependencies which are in need of some updating, but are otherwise harmless and in a separate go module not used by go-ipfs itself.

Reply:
Oh right, I missed that this was not `./go.mod`. Even the test dependencies don't seem to depend on the library in question though:
```
 ~/src/go/src/github.com/ipfs/go-ipfs/test/dependencies | master >
❯ go mod why github.com/microcosm-cc/bluemonday
# github.com/microcosm-cc/bluemonday
(main module does not need package github.com/microcosm-cc/bluemonday)